randovania/data/json_data/prime3.txt

```diff
@@ -3631,7 +4483,7 @@ Asset id: 11846340785047609812
       Trivial
 
 > Dock to Processing Access; Heals? False
-  * Normal Door to Processing Access/Dock to Scrapvault
+  * Plasma Door to Processing Access/Dock to Scrapvault
   > Room Center
       Trivial
 
```

Comment:
Revisit the nodes below. I think some of these need to be changed to morph doors, and/or change the nodes they connect to and the requirements needed.